### PR TITLE
fix(dropdown): add placeholder for label inside

### PR DESCRIPTION
**Pull request by EmelieLitwin**
**Describe pull-request**
Dropdown with label inside is not rendering the placeholder text.

**Solving issue**
Github issue https://github.com/scania-digital-design-system/sdds/issues/440 and Azure ticket [AB#2321](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2321)

**How to test**
Add description how to test if possible

Go to storybook
Check dropdown label inside and see if it looks correct

### CODE REVIEW

**Comment by matyldamc** [comment]
@EmelieLitwin the placeholder text is now rendering in Storybook. However, what state is it meant to show? Because the default state for the "label inside" variant is just the dropdown with the label inside as placeholder text (see attached). 
Screenshot 2022-10-03 at 16 12 1

File: components/src/components/dropdown/dropdown.scss [hunk 1]
```diff
@@ -149,13 +150,21 @@
   &.sdds-dropdown-lg {
     .sdds-dropdown-toggle {
       height: 56px;
+
+      :has(.sdds-dropdown-label-inside) {
```
Comment (mJarsater) [inline components/src/components/dropdown/dropdown.scss:154]: The :has pseudo selector isn't supported on Firefox, so maybe try and solve this in some other way?  We had a similar issue on this PR: https://github.com/scania-digital-design-system/sdds/pull/485

Comment (EmelieLitwin) [reply components/src/components/dropdown/dropdown.scss:154]: Thank you @mJarsater !

**Review by theJohnnyMe — APPROVED** [review]
Looks good to me

**Pull request closed without merging**